operator/config/watcher/watcher.yaml

```diff
@@ -1,18 +1,18 @@
 apiVersion: operator.kyma-project.io/v1alpha1
 kind: Watcher
 metadata:
-  name: controller-manager-watcher
+  name: watcher
   namespace: system
   labels:
     "operator.kyma-project.io/managed-by": "lifecycle-manager"
 spec:
   labelsToWatch:
-    "watched-by": "lifecycle-manager"
+    "operator.kyma-project.io/watched-by": "lifecycle-manager"
   field: "spec"
   serviceInfo:
-    name: event-service
+    name: lifecycle-manager-event-service
     port: 8082
-    namespace: system
+    namespace: kcp-system
```

Comment:
Please double check if this namespace really needed, otherwise drop it. I checked the code, basically there is no real usage just print out a hostname. The reason is this namespace is not controlled by Kustomize overwrite by default, we need to add additional patch to keep it consisten.
```
 func destinationHost(serviceName, serviceNamespace string) string {
	return fmt.Sprintf("%s.%s.svc.cluster.local", serviceName, serviceNamespace)
}

Reply:
The namespace is needed because when configuring the destination rule in the virtual service, we need to include the namespace in order for the virtual service to know how to route the traffic to the service in case of different namespace, see:
https://github.com/istio/istio.io/blob/4b89c302aea85cf7041db7617984aadcb08599b4/content/en/docs/concepts/traffic-management/index.md?plain=1#L240-L252

Reply:
Resolving after huddle 👍